### PR TITLE
Rename getPoint to getPointPixelPacket and bind vps_getpoint to getPoint

### PR DESCRIPTION
**Pull request by stuart12**
As getPointPixelPacket does much more than just call vips_getpoint, rename
getPoint to getPointPixelPacket.

Bind vps_getpoint to getPoint.

### CODE REVIEW

File: src/test/java/com/criteo/vips/VipsImageImplTest.java [hunk 1]
```diff
@@ -690,4 +691,37 @@ public void TestHistFindNdimMax1() throws IOException, VipsException {
...
     }
+
+    @Test
+    public void TestGetPoint() throws IOException, VipsException {
+        ByteBuffer buffer = VipsTestUtils.getDirectByteBuffer("in_vips.jpg");
+        VipsImageImpl img = new VipsImageImpl(buffer, buffer.capacity());
+
+        double[] point = img.getPoint(33, 8);
+        assertEquals(3, point.length);
+        double correct[] = { 0, 84, 219 };
```
Comment (dbouron) [inline src/test/java/com/criteo/vips/VipsImageImplTest.java]: Please rename to `expected`

File: src/test/java/com/criteo/vips/VipsImageImplTest.java [hunk 2]
```diff
@@ -690,4 +691,37 @@ public void TestHistFindNdimMax1() throws IOException, VipsException {
...
+
+        img.histFindNdim(10);
+        Max1Result r = img.max1();
+        assertEquals(239667.0, r.out);
+        assertEquals(0, r.x);
+        assertEquals(4, r.y);
+
+        double[] point = img.getPoint(r.x, r.y);
+        assertEquals(10, point.length);
+        double correct[] = { 0, 0, 15, 226, 1966, 10993, 101474, 239667, 61531, 1693 };
```
Comment (dbouron) [inline src/test/java/com/criteo/vips/VipsImageImplTest.java]: Same

File: src/test/java/com/criteo/vips/VipsImageImplTest.java (hunk 1 above)
Comment (stuart12) [reply src/test/java/com/criteo/vips/VipsImageImplTest.java]: done

File: src/test/java/com/criteo/vips/VipsImageImplTest.java (hunk 2 above)
Comment (stuart12) [reply src/test/java/com/criteo/vips/VipsImageImplTest.java]: done

**Pull request merged**